snscrape/modules/instagram.py

```diff
@@ -87,13 +88,22 @@ def _response_to_items(self, response):
 			username = node['node']['owner']['username'] if 'username' in node['node']['owner'] else ''
 			usernameQuery = '?taken-by=' + username
 			cleanUrl = f'https://www.instagram.com/p/{code}/'
+			displayUrl = node['node']['display_url']
+			mediaUrls = []
+			mediaUrls.append(displayUrl)
+			if node['node'].get('edge_sidecar_to_children') != None:
+				mediaUrlsData = node['node']['edge_sidecar_to_children']['edges']
+				for mediaData in mediaUrlsData:
+					if mediaData['node']['__typename'] == 'GraphImage':
+						mediaUrls.append(mediaData['node']['display_url'])
```

Comment:
Does Instagram return any further information, e.g. resolution or distinction between images and videos? If so, it would be nice to also extract those, similar to how it's implemented on the Twitter scraper.

Reply:
Here's an example, I'll add `IsVideo` and the dimensions, anything else for it ??

https://paste.sr.ht/~dallas/667399000c32d842fbfe3d1f4744542d44261c1a

Reply:
For consistency, rather than `isVideo`, I'd prefer it to be structured the same as on the Twitter module, i.e. a `Medium` class with `Photo` and `Video` subclasses which then contain the appropriate attributes, and then a `media` attribute on `InstagramPost` of type `typing.Optional[typing.List[Medium]]`.

`accessibilityCaption` might be nice. I don't see anything else in that example.